js/app/modules/arrangement/accordion.js

```diff
@@ -32,6 +32,8 @@ const Accordion = new Module.Class({
         let label = new Gtk.Button({
             label: card.model.title,
         });
+        // FIXME: Is there a less intrusive way to do this?
+        label.get_child().halign = Gtk.Align.START;
```

Comment:
I would just make you own label in this case and pack it in button directly (all gtk button is doing internally i think)